packages/generator-joplin/generators/app/templates/api/JoplinViewsEditor.d.ts

```diff
@@ -1,5 +1,22 @@
 import Plugin from '../Plugin';
-import { ActivationCheckCallback, ViewHandle, UpdateCallback } from './types';
+import { ActivationCheckCallback, ViewHandle, UpdateCallback, EditorPluginCallbacks } from './types';
+export interface EditorPluginProps {
+    /** The ID of the window to show the editor plugin. Use `undefined` for the main window. */
+    windowId: string | undefined;
```

Comment:
Any reason not to make the property optional in this case? (I know we have this rule to avoid optional properties but for user-facing code it's different as we want to make things easier)